src/styles/block/conlust-form.scss

```
@@ -0,0 +1,51 @@
+.consult-form {
+  flex-basis: 40%;
+
+  @media screen and(max-width: $breakpointMiddle) {
+    flex-basis: 100%;
+    margin-top: 32px;
+  }
+
+  &__input {
+    display: block;
+    width: 100%;
+    margin-bottom: 16px;
+    padding: 17px;
+    border: 2px solid $mainBackgrounColor;
+    border-radius: 8px;
+    background-color: $inputBGColor;
+
+    &:hover {
+      border-color: $blueColor;
+    }
+
+    &::placeholder {
+      color: $placeHolderColor;
+      text-transform: capitalize;
+      font-size: 14px;
+    }
+  }
+
+  &__button {
+    display: block;
+    width: 100%;
+    color: $whiteColor;
+    background-color: $blueColor;
+    margin-top: 32px;
+    padding: 17px;
+    border: 1px solid $blueColor;
+    border-radius: 8px;
+    font-size: 16px;
+    font-weight: 700px;
+    line-height: 14px;
+    letter-spacing: 1.2px;
+    text-transform: capitalize;
+    transition: all $transitionSpeed;
```

do not use all
that's not speed, that's duration
```suggestion
    transition: css-property $duration;
```